packages/cli/lib/DevServerManager.js

```diff
@@ -75,6 +81,11 @@ class DevServerManager {
             debug,
             onUploadRequired,
             promptUser,
+            logger,
+            urls: {
```

Comment:
I thought we initially needed the `env`, but we were exchanging that for the api and web urls using the legacy `httpClient.getRequestOptions` method.  To get rid of the need for that I found these two methods to get the urls.  It seems to be the same as what we want.

Reply:
Just fyi, `getRequestOptions` was replaced with [getAxiosConfig](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/http/getAxiosConfig.ts). But that being said, I like what you have here better than using getAxiosConfig 👍